test/integration/controllers/review.spec.js

```diff
@@ -0,0 +1,177 @@
+const { serverInit, serverCleanup } = require('~/test/setup')
+const { REVIEW_NOT_FOUND, UNAUTHORIZED } = require('~/consts/errors')
+const { expectError } = require('~/test/helpers')
+const testUserAuthentication = require('~/utils/testUserAuth')
+
+const endpointUrl = '/reviews/'
+
+let testReview, accessToken
+const nonExistingReviewId = '63bed9ef260f18d04ab15da2'
+
+describe('Review controller', () => {
+  let app, server
+
+  beforeAll(async () => {
+    ;({ app, server } = await serverInit())
+  })
+
+  afterAll(async () => {
+    await serverCleanup(server)
+  })
+
+  describe(`POST ${endpointUrl}`, () => {
+    it('should throw UNAUTHORIZED', async () => {
+      const response = await app.post(endpointUrl).send({
+        comment: 'Good mentor and learning program',
+        rating: 5,
+        targetUserId: '63bed43c51ee69a0d4c5ff92',
+        targetUserRole: 'tutor',
+        offerId: '63bed445cf00bcbdcdb4e648'
+      })
+
+      expectError(401, UNAUTHORIZED, response)
+    })
+
+    it('should create a review', async () => {
+      accessToken = await testUserAuthentication(app)
+
+      const response = await app.post(endpointUrl).set('Authorization', `Bearer ${accessToken}`).send({
```

Comment:
you keep repeating the same comment over and over again, move it to the top

Reply:
fixed